lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -0,0 +1,534 @@
+  processingEndDateTime: dateString,
+  lastUpdateDateTime: dateString,
+  timeToArchive: 0,
+  productionDateTime: dateString,
+  timestamp: Date.now(),
+  createdAt: Date.now() - 200 * 1000,
+  updatedAt: Date.now(),
+});
+
+let collectionsModel;
+let executionsModel;
+let granulesModel;
+
+const testDbName = `data_migration_2_${cryptoRandomString({ length: 10 })}`;
+
+test.before(async (t) => {
+  await s3Utils.createBucket(bucket);
+  process.env.stackName = cryptoRandomString({ length: 10 });
+  process.env.system_bucket = bucket;
+
+  process.env.CollectionsTable = cryptoRandomString({ length: 10 });
+  process.env.ExecutionsTable = cryptoRandomString({ length: 10 });
+  process.env.GranulesTable = cryptoRandomString({ length: 10 });
+
+  collectionsModel = new Collection();
+  await collectionsModel.createTable();
+
+  executionsModel = new Execution();
+  await executionsModel.createTable();
+
+  granulesModel = new Granule();
+  await granulesModel.createTable();
+
+  t.context.granulePgModel = new GranulePgModel();
+  t.context.filePgModel = new FilePgModel();
+
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+});
+
+test.beforeEach(async (t) => {
+  const collectionPgModel = new CollectionPgModel();
+  const testCollection = fakeCollectionRecordFactory();
+
+  const collectionResponse = await collectionPgModel.create(
+    t.context.knex,
+    testCollection
+  );
+  t.context.testCollection = testCollection;
+  t.context.collectionCumulusId = collectionResponse[0];
+
+  const executionPgModel = new ExecutionPgModel();
+  const testExecution = fakeExecutionRecordFactory();
+
+  const executionResponse = await executionPgModel.create(
+    t.context.knex,
+    testExecution
+  );
+  t.context.testExecution = testExecution;
+  t.context.executionCumulusId = executionResponse[0];
+
+  t.context.testGranule = generateTestGranule(testCollection, testExecution.arn);
+});
+
+test.afterEach.always(async (t) => {
+  await t.context.knex(tableNames.files).del();
+  await t.context.knex(tableNames.granules).del();
+  await t.context.knex(tableNames.collections).del();
+  await t.context.knex(tableNames.executions).del();
+});
+
+test.after.always(async (t) => {
+  await granulesModel.deleteTable();
+  await collectionsModel.deleteTable();
+  await executionsModel.deleteTable();
+
+  await s3Utils.recursivelyDeleteS3Bucket(bucket);
+
+  await destroyLocalTestDb({
+    ...t.context,
+    testDbName,
+  });
+});
+
+test.serial('migrateGranuleRecord correctly migrates granule record', async (t) => {
+  const {
+    collectionCumulusId,
+    executionCumulusId,
+    granulePgModel,
+    knex,
+    testGranule,
+  } = t.context;
+
+  await migrateGranuleRecord(testGranule, knex);
+
+  const record = await granulePgModel.get(knex, {
+    granule_id: testGranule.granuleId,
+    collection_cumulus_id: collectionCumulusId,
+  });
```

Comment:
```suggestion
  const [granuleCumulusId] = await migrateGranuleRecord(testGranule, knex);

  const record = await granulePgModel.get(knex, {
    cumulus_id: granuleCumulusId
  });
```

Reply:
So we can look up a granule by either `granuleId` and `collectionCumulusId` OR `granuleCumulusId`?

Reply:
[`bc6cc2b` (#2051)](https://github.com/nasa/cumulus/pull/2051/commits/bc6cc2bd0d21127123f6a3bbe6cc5477bd3579fc#diff-91b3c5306441fb2380946f4cabdfa533f0143379d13bebd235dc5cd9af8d8586R53-R56)

Reply:
Yeah, you can look up any PG record either using its unique column(s) (the combo of `collectionCumulusId` and `granuleId` have a unique constraint for granules) or its primary key, which in this case is the `cumulus_id` column (and is for all of our tables)

Reply:
This change isn't required, but it doesn't look like it was addressed here